datadog_checks_downloader/datadog_checks/downloader/data/repo/metadata/previous/.gitignore

```diff
@@ -0,0 +1,5 @@
+# This file is here, so that we can add this initially empty directory as part
```

Comment:
We want to make these empty default directories when we install, so that we will download all files (TUF and in-toto metadata, as well as wheels) in there later, you see?

Reply:
I'm not sure I get it. Can't we create them on the fly ?